code/main/core/identityModule.brs

```diff
@@ -88,7 +88,7 @@ function _adb_IdentityModule(configurationModule as object) as object
                 return invalid
             end if
 
-            url = _adb_buildEdgeRequestURL(configId, _adb_generate_UUID(), edgeDomain)
+            url = _adb_buildEdgeRequestURL(configId, _adb_generate_UUID(), "", edgeDomain)
```

Comment:
I would prefer passing "/ee/v1/interact" instead of empty string. Can you declare an internal constant for default path and pass it to this function?

Reply:
Yeah, it makes sense. I can make the default sting a constant variable.